README.md

```diff
@@ -68,19 +68,13 @@ To import an individual component (for example, a button), add the line:
 ```
 
 
-To resolve your `@import` declarations you should add `node_modules` to your [Sass include paths](https://github.com/sass/node-sass#includepaths) or [assets paths](http://guides.rubyonrails.org/asset_pipeline.html#search-paths) in Ruby.
+To resolve your `@import` declarations you should add `node_modules` to your [Sass include paths](https://github.com/sass/node-sass#includepaths) or [assets paths](http://guides.rubyonrails.org/asset_pipeline.html#search-paths) in Ruby if you don't have it set up already.
```

Comment:
Could we be even more explicit?

"When you `npm install` our packages they will install to the `node_modules` directory, by default Sass does not look in the `node_modules` directory when resolving `@import` declarations."

Might be too many words, not sure. 🤷‍♂️